### PR TITLE
Implement UX improvements

### PR DESCRIPTION
**Pull request by sigalsax**
### What does this PR do?
Implements second round of UX feedback

Executable can be found here:
[conjur-cli-round-2.tar.gz](https://github.com/cyberark/conjur-api-python3/files/5899888/conjur-cli-round-2.tar.gz)

### CODE REVIEW

File: conjur/errors.py [hunk 1]
```diff
@@ -25,4 +25,13 @@ def __init__(self, message="Error: Failed to run command to completion."):
         super().__init__(self.message)
 
 class MissingRequiredParameterException(Exception):
-    """ Exception for when user does not input a required paramter """
+    """ Exception for when user does not input a required parameter """
```
Comment (InbalZilberman) [inline conjur/errors.py:28]: @sigalsax is this a remark or a message? or???

File: conjur/init/init_controller.py [hunk 2]
```diff
@@ -58,6 +62,10 @@ def get_server_certificate(self):
                 # pylint: disable=raise-missing-from
                 raise RuntimeError("Error: URL is required")
 
+        # Chops off the '/ if supplied by the user to avoid a server error
```
Comment (InbalZilberman) [inline conjur/init/init_controller.py:65]: nice!

File: conjur/init/init_controller.py [hunk 3]
```diff
@@ -58,6 +62,10 @@ def get_server_certificate(self):
...
+        if self.conjurrc_data.appliance_url.endswith('/'):
+            print(self.conjurrc_data.appliance_url[:-1])
```
Comment (InbalZilberman) [inline conjur/init/init_controller.py:67]: why do we print?

File: conjur/policy/policy_controller.py [hunk 4]
```diff
@@ -23,5 +28,11 @@ def load(self):
...
         """
-        result = self.policy_logic.run_action(self.policy_data)
-        sys.stdout.write(result+'\n')
+        try:
+            result = self.policy_logic.run_action(self.policy_data)
+            sys.stdout.write(result+'\n')
+        except requests.exceptions.HTTPError as http_error:
+            if http_error.response.status_code == 422:
+                raise InvalidFormatException(f"{http_error}. The policy was empty or its contents "
+                                             "were not in valid YAML.") from http_error
```
Comment (InbalZilberman) [inline conjur/policy/policy_controller.py:37]: How about - or its content was not a valid YMAL.

**Review by InbalZilberman — COMMENTED** [review]
left few comments

File: conjur/errors.py (hunk 1 above)
Comment (sigalsax) [reply conjur/errors.py:28]: this is a comment (:

File: conjur/init/init_controller.py (hunk 3 above)
Comment (sigalsax) [reply conjur/init/init_controller.py:67]: great catch this shouldn't have made it to the PR

File: conjur/policy/policy_controller.py (hunk 4 above)
Comment (sigalsax) [reply conjur/policy/policy_controller.py:37]: I think its better' or its content was not in valid YAML' or 'not in valid YAML syntax'?

Comment (InbalZilberman) [reply conjur/policy/policy_controller.py:37]: We need Shuli approval here.. how do you want to handle it?
BTW, how come Shuli is not a user in this repo?

**Review by InbalZilberman — APPROVED** [review]
small comment. other than that look good!

File: conjur/policy/policy_controller.py (hunk 4 above)
Comment (sigalsax) [reply conjur/policy/policy_controller.py:37]: I will send Shuli a message. Don't know why

Comment (sigalsax) [reply conjur/policy/policy_controller.py:37]: Do I have the power to add her?

Comment (shulifink) [reply conjur/policy/policy_controller.py:37]: @sigalsax  - approved.
Weird that you can't tag me.

**Review by shulifink — COMMENTED** [review]
approved

**Pull request merged**